src/sass/styles.scss

```diff
@@ -259,3 +265,32 @@ $border-thin: 1px solid $color-mid-light !default;
 .no-border-top {
   border-top: none !important;
 }
+
+.snapshot-creation-modal {
```

Comment:
This seems not be generic enough for the main file, maybe into _snapshots.scss or a new file?

Reply:
moved